src/lib/services/s3.js

```diff
@@ -61,7 +61,22 @@ const download = async (key, destination) => {
  */
 const getHead = key => callS3MethodWithKey('headObject', key);
 
+/**
+ * Gets a signed URL for a particular file
+ * @param {String} filepath
+ * @param {String} operation
+ */
+const getSignedUrl = (filepath, operation = 'getObject', expiry = 60) => {
```

Comment:
Not sure this is used any more?

Reply:
You're right. I was keeping it on the off chance it is useful in the future but I'll take it out